tests/unit/modules/test_pyroute2.py

```diff
@@ -0,0 +1,12 @@
+
+from pyroute2 import IPDB
+from tests.support.unit import TestCase
+from salt.beacons.network_settings import ATTRS
```

Comment:
Absolute import will fail and I usually would avoid this. Add at the beginning the following, if you want still stick to it:
```python
from __future__ import absolute_import
```